tests/x11/remote_desktop/vino_screensharing_available.pm

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+#
+#
+# Summary: Test if the Screen Sharing GNOME Desktop functionality works
+#   We currently need to install vino package for that
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
+
+use base "x11test";
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    assert_and_click 'menu', 'left', 90;
+    assert_and_click 'settings';
+    assert_screen 'all_settings';
+    type_string 'shar';
+    assert_and_click 'sharing';
+    assert_screen [qw(without_screensharing with_screensharing)];
+    if (match_has_tag 'without_screensharing') {
+        record_info 'vino missing', 'After the installation the screen sharing is not available - vino is missing.';
+        send_key 'ctrl-q';
+
+        record_info 'Install vino', 'We are going to install vino.';
+        mouse_hide;
+        ensure_installed 'vino';
+
+        handle_logout;
+        handle_login;
+
+        assert_and_click 'menu', 'left', 90;
+        assert_and_click 'settings';
+        assert_screen 'all_settings';
+        type_string 'shar';
+        assert_and_click 'sharing';
+
+        assert_screen 'with_screensharing';
+        record_info 'vino present', 'Vino and the screen sharing are present';
+
+    }
+    elsif (match_has_tag 'with_screensharing') {
+        record_info 'vino present', 'Vino and the screen sharing are present';
```

Comment:
This line and the one above in L52 are just the same so actually I think the whole if-block should not be necessary. If you extract this line (or delete it because its a bit too verbose in comparison to our common test style) and then delete the tag for 'with_screensharing' in L33 you can just get rid of the if. WDYT?